admin/src/templates/export_dhis.html

```diff
@@ -0,0 +1,38 @@
+<div class="col-sm-12">
+  <p translate>export.dhis.description</p>
+  <p ng-show="!dataSets" class="text-warning" translate>export.export.dhis.unconfigured</p>
+
+
+  <div ng-show="dataSets">
+    <div class="form-group">
+      <label for="dataset" translate>export.dhis.dataset.label</label>
+      <select id="dataset" class="form-control small" ng-model="selected.dataSet">
```

Comment:
![image](https://user-images.githubusercontent.com/583851/73260075-10870880-41c1-11ea-80cd-427b5e983d65.png)

Using the `small` class makes the text cut off for me.